operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -1054,12 +1054,22 @@ function wdtHandleOnlineUpdate() {
   createFolder "${DOMAIN_HOME}/lib" "This is the './lib' directory within directory 'domain.spec.domainHome'." || exitOrLoop
   for file in $(sort_files ${IMG_ARCHIVES_ROOTDIR} "*.zip")
     do
-        # expand the archive domain libraries to the domain lib
+        # expand the archive domain libraries to the domain lib, 11 is caution when zip entry doesn't exists
         cd ${DOMAIN_HOME}/lib || exitOrLoop
-        ${JAVA_HOME}/bin/jar xf ${IMG_ARCHIVES_ROOTDIR}/${file} wlsdeploy/domainLibraries/
+        unzip -jo ${IMG_ARCHIVES_ROOTDIR}/${file} wlsdeploy/domainLibraries/*
+        if [ $? -ne 0 && $? -ne 11 ] ; then
+          trace SEVERE  "Domain Source Type is FromModel, error in extracting domainLibraries
+          ${IMG_ARCHIVES_ROOTDIR}/${file}"
+          exitOrLoop
+        fi
 
-        if [ $? -ne 0 ] ; then
-          trace SEVERE  "Error extracting domain lib '${IMG_ARCHIVES_ROOTDIR}/${file}'"
+        # expand the domain bin, in update case user may only update a file in the domainBin archive, 11 is caution when
+        # zip entry doesn't exists
+        cd ${DOMAIN_HOME}/bin || exitOrLoop
+        unzip -jo ${IMG_ARCHIVES_ROOTDIR}/${file} wlsdeploy/domainBin/*
+        if [ $? -ne 0 && $? -ne 11 ] ; then
+          trace SEVERE  "Domain Source Type is FromModel, error in extracting domainBin
+          ${IMG_ARCHIVES_ROOTDIR}/${file}"
```

Comment:
I didn't know that you could break a string across multiple lines like this, but don't have access to a shell at the moment to try it out.  Cool. Will it ignore the indent spaces?

OTOH It seems a little subtle and easy to break.  Alternatively, I think trace supports any number of arguments, so you could do this instead:
```
 trace SEVERE  "Domain Source Type is FromModel, error in extracting domainBin" \
          " ${IMG_ARCHIVES_ROOTDIR}/${file}"
```

Reply:
Yes, it will, shell knows it's continuation since the double quote has not been ended. I can change it for the easier to read convention